sdk/eventhubs/azure-messaging-eventhubs-stress/README.md

```diff
@@ -0,0 +1,300 @@
+# Azure Event Hubs Stress test client library for Java
+
+Represents stress tests for Event Hubs client.
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk_link], version 8 or later.
+- [Maven][maven]
+- [Docker][docker]
+- [Kubectl][kubectl]
+- [Helm][helm]
+- [Azure CLI][azure_cli]
+- [Powershell 7.0+][powershell]
+
+Make sure you have permission to access subscription `Azure SDK Developer Playground` and resource group `rg-stress-cluster-test`.
+
+```shell
+# Authenticate to Azure
+az login
+
+# Download the kubeconfig for the cluster (creates a 'context' named 'stress-test')
+az aks get-credentials --subscription "Azure SDK Developer Playground" -g rg-stress-cluster-test -n stress-test
+```
+
+### Deploy to cluster
+
+Build out the jar package:
+
+```shell
+cd <current project path>
+mvn clean install
+```
+
+Update the `namespace` field in `Chart.yaml` file. It is suggested to use your own alias to avoid conflict.
```

Comment:
FYI for local deployment with the script, we already override the namespace based on the the shell `$USER`.

Reply:
Oh, that's why I change namespace value in `Chart.yaml`, it still deployment on the namespace of my alias. I can note down this information to avoid misunderstanding.

Reply:
Yeah it looks like we don't log anything when we're selecting the namespace and repository overrides with username. I'll add a log statement which might help in some cases.